pkgs/applications/misc/oneko/default.nix

```diff
@@ -0,0 +1,48 @@
+{ stdenv, fetchurl, xorg, x11 }:
+
+let version = "1.2.sakura.5";
```

Comment:
If you move version inside the attribute set it can be overridden (since it's a recursive set already, the let here doesn't really make sense).